packages/xo-server-test/src/vm/index.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env jest */
+
+import config from '../_config'
+import xo from '../_xoConnection'
+
+describe('vm', () => {
+  describe('.set() :', () => {
+    it('set properties to null', async () => {
+      await xo.createTempServer(config.servers.default)
+      const vm = await xo.createTempVm({
+        coresPerSocket: 1,
+        cpuCap: 1,
+        name_label: 'XO Test',
+        template: config.templates.templateWithoutDisks,
```

Comment:
`name_label` and `template` could probably have default values in `createTempVm`.